### PR TITLE
PMM-4459 Disable query examples.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-4459

depends on:

- [x] https://github.com/percona/pmm/pull/425
- [x] https://github.com/percona/pmm-agent/pull/115
- [x] https://github.com/percona/pmm-admin/pull/58
- [x] https://github.com/percona/qan-app/pull/322

FB: https://github.com/Percona-Lab/pmm-submodules/pull/441

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-441-f2d1805
client docker - perconalab/pmm-client-fb:PR-441-f2d1805
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-441-f2d1805.tar.gz

File: .gitmodules [hunk 1]
```diff
@@ -47,23 +47,23 @@
 [submodule "pmm"]
 	path = sources/pmm/src/github.com/percona/pmm
 	url = https://github.com/percona/pmm
-	branch = PMM-2.0
+	branch = PMM-4459-disable-query-example
```
Comment (AlekSi) [inline .gitmodules]: Merged

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-441-9a07674
client docker - perconalab/pmm-client-fb:PR-441-9a07674
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-441-9a07674.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-441-eaa5640
client docker - perconalab/pmm-client-fb:PR-441-eaa5640
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-441-eaa5640.tar.gz

**Pull request closed without merging**